flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/plan/nodes/physical/stream/StreamExecExchange.scala

```diff
@@ -32,7 +44,10 @@ class StreamExecExchange(
     relNode: RelNode,
     relDistribution: RelDistribution)
   extends CommonPhysicalExchange(cluster, traitSet, relNode, relDistribution)
-  with StreamPhysicalRel {
+  with StreamPhysicalRel
+  with StreamExecNode[BaseRow] {
+
+  private val DEFAULT_MAX_PARALLELISM = 1 << 7
```

Comment:
too small?

Reply:
The variable's name is confusing, it's not max parallelism of operators, but max number of key-groups. Maybe it's better to use `StreamGraphGenerator.DEFAULT_LOWER_BOUND_MAX_PARALLELISM` or `KeyGroupRangeAssignment.DEFAULT_LOWER_BOUND_MAX_PARALLELISM`

Reply:
+1 . I'm in favor of the later one which is also used in [`KeyedStream.java#134` ](https://github.com/apache/flink/blob/ddba1b69f43cbb885e178dfaafa120f1fe196a13/flink-streaming-java/src/main/java/org/apache/flink/streaming/api/datastream/KeyedStream.java#L134) and has a clear javadoc description.